undate/undate.py

```diff
@@ -63,10 +64,14 @@ def __init__(
             formatter = BaseDateFormat.available_formatters()[self.DEFAULT_FORMAT]()
         self.formatter = formatter
 
+        self.label = label
```

Comment:
Question for the Python Gurus: `label` and a few other fields are defined as class variables and then set as instance variables in the init method, right? In `UndateInterval` the fields are all instance variables. Is there a reason for the difference in implementation? Is there a benefit to either approach? Just curious :)

Reply:
Good question! We should probably be consistent about that.

You can declare instance variables on the class for default values and for documentation purposes, which is what I think I usually do. I don't know if there is a best practice about whether they should or should not be declared on the class, though.